.golangci.yml

```diff
@@ -10,106 +10,74 @@ linters:
   enable:
     - asciicheck
     - bidichk
-    #    - bodyclose <- needs go 1.18 support
-    # - contextcheck <- needs go 1.18 support
-    #    - cyclop
     - depguard
-    #    - dogsled <- disabled because we would like to participate in the iditerod
-    # - dupl <- really just catches cli stub text
     - durationcheck
     - errcheck
     - errname
-#    - errorlint <- later
-#    - exhaustive <- too picky
-    #    - exhaustivestruct <- disabled because we don't want to use every element of every struct
     - exportloopref
-    #    - forbidigo <- forbids fmt.Print* by default
     - forcetypeassert
-    #    - funlen <- some of our functions are longer than this by necessity
-    #    - gci <- questionable utility
-    #    - gochecknoglobals <- disabled because we want some globals
-    #    - gochecknoinits <- disabled because we want to use init()
-    #    - gocognit <- disabled for same reason as gocyclo
     - goconst
-    # - gocritic <- re enable later to catch capitalized local variables
-    #    - gocyclo <- disabled because it won't work with how we cosmos
-    # - godot
-    #    - godox <- detects TODO/BUG/FIXME and we may wantnthis later but isn't appropriate now
-    #    - goerr113 <- disabled due to lack of comprehension
-    - gofmt
-    # - gofumpt <- disabled https://github.com/osmosis-labs/osmosis/issues/1309
+    - gocritic # <- re enable later to catch capitalized local variables
+    - gofumpt
     - goheader
-    - goimports
-    #    - gomoddirectives <- disables replaces
     - gomodguard
     - goprintffuncname
-#    - gosec <- triggers too much for this round and should be re-enabled later
+    - gosec # <- triggers too much for this round and should be re-enabled later
     - gosimple # specializes in simplifying code, subset of staticcheck binary.
     - govet
-    #   - ifshort <- questionable value, unclear if globally helping readability.
     - importas
     - ineffassign
-    #    - ireturn <- disabled because we want to return interfaces
-    #    - lll <- disabled as it is a bit much.  Maybe we have a desired limit?
     - makezero
     - misspell
     - nakedret
-#    - nestif <- ?
-    # - nilerr <- needs go 1.18 support
     - nilnil
-    #    - nlreturn <- disabled as it doesn't auto-fix something simple and doesn't add anything useful
-    # - noctx <- needs go 1.18 support
-    #    - nolintlint <- disabled because we use nolint in some places
+    - nolintlint # <- disabled because we use nolint in some places
```

Comment:
Lets delete this?

Nolints are perfectly fine to have in the codebase.

Reply:
nolintlint just forces us to 

//nolint:whatever per-line, I'd prefer to keep it ? 

Basically it is an enforcer of //nolint:whatever hygene.

Reply:
oh gothca